photon-client/src/views/CamerasView.vue

```diff
@@ -486,9 +493,96 @@ export default {
+
+            var x_coord = 0.0;
+            var y_coord = 0.0;
+            var x_idx = 0;
+            var y_idx = 0;
+            var start_x = 0;
+            var start_y = 0;
+
+            var annotation = num_x + " x " + num_y + " | " + patternSize + "in "
+
+            if(isCheckerboard){
+              ///////////////////////////////////////////
+              // Checkerboard Pattern
+
+              start_x = paper_x/2.0 - (num_x * patternSize)/2.0;
+              start_y = paper_y/2.0 - (num_y * patternSize)/2.0;
+
+              for(y_idx = 0; y_idx < num_y; y_idx++){
+                for(x_idx = 0; x_idx < num_x; x_idx++){
+
+                  x_coord = start_x + x_idx * patternSize;
+                  y_coord = start_y + y_idx * patternSize;
+                  if((x_idx + y_idx) % 2 == 0){
+                    doc.rect(x_coord, y_coord, patternSize, patternSize, "F");
+                  }
+                }
+              }
+
+            } else {
+              ///////////////////////////////////////////
+              // Assymetric Dot-Grid Pattern
+              // see https://github.com/opencv/opencv/blob/b450dd7a87bc69997a8417d94bdfb87427a9fe62/modules/calib3d/src/circlesgrid.cpp#L437
+              // as well as FindBoardCornersPipe.java's Dotboard implementation
+
+              start_x = paper_x/2.0 - ((2*(num_x-1) + (num_y-1) % 2) * patternSize)/2.0;
+              start_y = paper_y/2.0 - (num_y-1 * patternSize)/2.0;
+
+              // Dot Grid Pattern
+              for(y_idx = 0; y_idx < num_y; y_idx++){
+                for(x_idx = 0; x_idx < num_x; x_idx++){
+                  x_coord = start_x + (2*x_idx + y_idx % 2) * patternSize;
+                  y_coord = start_y + y_idx * patternSize;
+                  doc.circle(x_coord, y_coord, patternSize/4.0, "F");
+                }
+              }
+            }
+
+            ///////////////////////////////////////////
+            // Draw a fixed size inch ruler pattern to 
+            // help users debug their printers
+            var lineStartX = 1.0;
+            var lineEndX = paper_x - lineStartX;
+            var lineY = paper_y - 1.0;
+            doc.setFont('Prompt-Regular');
+            doc.setLineWidth(0.01);
+            doc.line(lineStartX, lineY, lineEndX, lineY);
+            var segIdx = 0;
+            for(var tickX = lineStartX; tickX <= lineEndX; tickX += 1.0){
+              doc.line(tickX, lineY, tickX, lineY + 0.25);
+              doc.text(String(segIdx) + (segIdx == 0 ? " in" : ""), tickX + 0.1, lineY + 0.25);
+              segIdx++;
+            }
+
+
+            ///////////////////////////////////////////
+            // Annotate what was drawn + branding
+            doc.setFont('MuseoModerno');
+            doc.text("PhotonVision", 1.0, 1.0, {maxWidth:(paper_x - 2.0)/2, align:"left"});
+            doc.setFont('Prompt-Regular');
+            doc.text(annotation, paper_x-1.0, 1.0, {maxWidth:(paper_x - 2.0)/2, align:"right"});
```

Comment:
Per https://github.com/PhotonVision/photonvision-branding/blob/master/PhotonVision%20Branding/Documentation/Photon%20Vision%20Branding%20Standards.pdf

Museo Moderno is restricted to logo usage only with Prompt for heading / title and Lato for body. This should be changed accordingly if possible.

B/W version of the logo w/ Museo Moderno are avaliable here:  https://github.com/PhotonVision/photonvision-branding/blob/master/PhotonVision%20Branding/Logos/Header/PNG/PhotonVision-Header-singleColor.png

Reply:
changed to real logo!